CleanWeather/Models/Weather.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Weather.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 12/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+struct Weather: Codable {
+    
+    enum SFsymbol: String {
```

Comment:
co to jest SFsymbol? ja nie mam pojęcia, nazwijmy to tak aby było wiadomo o co biega.